### ISSUE TITLE
bro.sh tries to use a malloc 

### ISSUE THREAD

**Issue by kraigu** [ncsa/bro-cluster-in-a-box-setup#4]
Line 58 (in current version) of bro.sh has a check for $malloc, but that variable is never set. This should probably be settable somewhere, or maybe remove that and force whatever malloc we think is best on RHEL7/CentOS7. (Is that tcmalloc?)

**Comment by JustinAzoff** [comment]
yeah, that was left over from the script I based it on.  I believe tcmalloc gets used automatically which is why I didn't notice, but forcing jemalloc would be even better.
